src/test/java/com/minecolonies/coremod/inventory/InventoryTest.java

```diff
@@ -42,6 +44,6 @@ public void addStackTest()
         final Item testItem = mock(Item.class);
         final ItemStack stuff = new ItemStack(testItem, 3);
         inventory.setInventorySlotContents(0, stuff);
-        //assertThat(inventory.getStackInSlot(0), is(stuff));
+        assertSame(inventory.getStackInSlot(0), stuff);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a message to this assertion. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2698)